app/jobs/pastes_cleanup_job.rb

```diff
@@ -4,7 +4,8 @@
 class PastesCleanupJob < ApplicationJob
   queue_as :default
 
-  def perform(paste)
-    paste.destroy
+  def perform(paste_id)
+    paste = Paste.where(id: paste_id)[0]
+    paste.destroy unless paste.nil?
```

Comment:
```suggestion
    paste&.destroy
```
no need to check for nil here

Reply:
Ah!